powersimdata/network/usa_tamu/constants/zones.py

```diff
@@ -20,12 +20,15 @@
     "loadzone2id",
     "loadzone2interconnect",
     "loadzone2state",
+    "mappings",
```

Comment:
I am kind of wondering since `mappings` is kind of special and will likely only be used in the `ModelImmutables` class if we should not remove from the `exports` list, and define the `mappings` attribute in the constructor of the `ModelImmutables` class through `self.mappings = mod.mappings`

Reply:
What is the idea behind the `exports` list? I can refactor however makes most sense with that original plan.

Reply:
It is to modify the standard definition of `dir` (see PEP 562) to allow easy look up in the `ModelImmutables` class doing `{a: getattr(mod, a) for a in dir(mod)}`